onnx_tf/handlers/backend/tile.py

```diff
@@ -8,22 +8,41 @@
 @onnx_op("Tile")
 @tf_func(tf.tile)
 class Tile(BackendHandler):
+  cast_map = {tf.uint16: tf.uint32}
+  #supported_types = [
+  #    tf.uint8, tf.uint32, tf.uint64, tf.int8, tf.int16, tf.int32, tf.int64,
+  #    tf.bfloat16. tf.float16, tf.float32, tf.float64, tf.complex64,
+  #    tf.complex128, tf.string, tf.bool
+  #]
```

Comment:
Please provide comments why this commented out.

Reply:
This is not needed, I will remove it.